src/sagemaker_pytorch_serving_container/transformer.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from __future__ import absolute_import
+
+import logging
+import traceback
+
+from six.moves import http_client
+from sagemaker_inference.transformer import Transformer
+from sagemaker_inference import content_types, environment, utils
+from sagemaker_inference.errors import BaseInferenceToolkitError, GenericInferenceToolkitError
+
+
+class PTTransformer(Transformer):
+    """Represents the execution workflow for handling pytorch inference requests
+    sent to the model server.
+    """
+    def __init__(self, default_inference_handler=None):
+        super().__init__(default_inference_handler)
+        self._context = None
+
+    def transform(self, data, context):
+        """Take a request with input data, deserialize it, make a prediction, and return a
+        serialized response.
+        Args:
+            data (obj): the request data.
+            context (obj): metadata on the incoming request data.
+        Returns:
+            list[obj]: The serialized prediction result wrapped in a list if
+                inference is successful. Otherwise returns an error message
+                with the context set appropriately.
+        """
+
+        try:
+            properties = context.system_properties
+            model_dir = properties.get("model_dir")
+            self.validate_and_initialize(model_dir=model_dir, context=self._context)
+
+            input_data = data[0].get("body")
```

Comment:
@waytrue17 I think this change should be incorporated to handle batch inference - https://github.com/aws/sagemaker-inference-toolkit/pull/108/files?